config/locales/en.yml

```diff
@@ -50,6 +50,10 @@ en:
         someone_invited_you: "%{their_name} (%{their_email}) has invited %{your_name} to create an account on GetYourRefund to help people with their taxes."
         subject: You've been invited to GetYourRefund
   case_management:
+
```

Comment:
a harmless extra newline. If you ever want to clean up the format of the translation files, you can run `i18n-tasks normalize` on the command line and it will alphabetize and organize all the keys.